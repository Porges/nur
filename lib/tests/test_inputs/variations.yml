---
version: "1.0"

default:
  after: [runString, runSh]
  run:
  - echo 'plain cmd'
  - sh: echo 'alternate cmd'

runString:
  run: echo 'simple run'

runSh:
  run:
    sh: echo 'run sh'
