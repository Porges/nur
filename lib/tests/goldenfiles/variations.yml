---
version: "1.0"

default:
  deps: [runString, runSh]
  run:
  - echo 'plain cmd'
  - sh: echo 'alternate cmd'

runString:
  run: echo 'simple run'

runSh:
  run:
    sh: echo 'run sh'
